app/policies/account_policy.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class AccountPolicy
```

Comment:
@puppe1990 we will call `AccountPolicy.new(current_user.account).enabled?` in the sidebar menu to hide elements from those accounts without bling flag.